labs/accessing-optimizely-event-data/README.md

```diff
@@ -0,0 +1,140 @@
+# Accessing Optimizely Enriched Event Data
+
+Optimizely [Enriched Events Export](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-export) gives you secure access to your Optimizely event data so you can analyze your experiment results with greater flexibility. The export includes a useful combination of events attributes:
+
+- Raw metadata (event names, user IDs, etc) that you pass to Optimizely, without additional processing
+- Enriched metadata that Optimizely adds such as experiment IDs, variation IDs, and session IDs
+
+For more information, see the Enriche Events [data specification](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification).
+
+This Lab contains `oevents`, a simple CLI tool for loading enriched event data.
+
+![oevents demo](img/demo.gif)
```

Comment:
Relative links don't work yet in labs. I added some instructions in the `CONTRIBUTING.md` section, but long story short, pre-fix this path with `https://raw.githubusercontent.com/optimizely/labs/master/labs/`

Example:
https://raw.githubusercontent.com/optimizely/labs/master/labs/feature-flags-ruby-sinatra/screenshots/app.png

From a feature flag lab:
https://github.com/optimizely/labs/blob/master/labs/feature-flags-ruby-sinatra/README.md

Once you merge the image should render and if we get the path wrong, we can fix it with a patch commit.

Reply:
ah, thank you--fixed